README.md

```diff
@@ -61,9 +62,18 @@ to **v1**
 
 ![Rollout stages](assets/rollback-stages.svg "Rollout stages from v1 to v2")
 
-## Try it out (locally)  <a id="try-out"></a>
+## Try it out (locally) <a id="try-out"></a>
 
-1. Check out this repository.
+1. Configure your service(s) to be selected for gradual rollouts by adding the
```

Comment:
1. [Label the Cloud Run services](https://cloud.google.com/run/docs/configuring/labels) (with label `x=y`) for them to be selected for gradual rollouts: